internal/gosmith/type.go

```diff
@@ -43,7 +43,7 @@ type Type struct {
 	utyp           *Type   // underlying type
```

Comment:
`ktyp` says `pointee` type. I decided it was intentional and left it alone. (see below)